app/controllers/credit_mutations_controller.rb

```diff
@@ -19,7 +19,7 @@ def create # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
       if @mutation.save
         NewCreditMutationNotificationJob.perform_later(@mutation) if Rails.env.production? || Rails.env.staging?
         format.html { redirect_to which_redirect?, flash: { success: 'Inleg of mutatie aangemaakt' } }
-        format.json { render json: @mutation, include: { user: { methods: %i[credit avatar_thumb_or_default_url] } } }
+        format.json { render json: @mutation, include: { user: { methods: %i[credit avatar_thumb_or_default_url minor insufficient_credit can_order] } } }
```

Comment:
`minor` and `insufficient_credit` should already have been in here, but were probably forgotten.